### DISCUSSION TITLE
Add advertisement with live stream

### DISCUSSION THREAD

**Discussion by mahendrakgujar88** [ant-media/Ant-Media-Server#6195] [Q&A]
How to add advertisement with live stream in ant media

**Comment by USAMAWIZARD** [comment]
Hi Mahendra , 
I hope you are doing well currently Advertisement support has not been added , we are still working on that feature and its on our roadmap for this year.

**Comment by lastpeony** [comment]
ID3 metadata and SEI on hls streams can be useful for displaying ads
Take a look at this blog post:
https://antmedia.io/id3-timed-metadata-to-create-interactive-hls-streams/

SEI support is recently merged to the master branch and it will be available after 2.9
If you urgently need it let us know.
